src/Components/Web.JS/Microsoft.AspNetCore.Components.Web.JS.npmproj

```diff
@@ -1,12 +1,15 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
-    <IsTestProject>true</IsTestProject>
```

Comment:
This project would not execute tests because the Arcade SDK unconditionally sets `$(IsTestProject)`, ignoring any prior setting